### PR TITLE
EMP-553: Refactored to move database related models to their own package

### PR DESCRIPTION
**Pull request by tanovellino**
EMP-553: Refactored to move database related models to their own package

## What

[Link to story](https://dsdmoj.atlassian.net/browse/EMP-553)

Describe what you did and why.

## Checklist

Before you ask people to review this PR:

- [ ] Tests should be passing: `./gradlew test`
- [ ] Github should not be reporting conflicts; you should have recently run `git rebase main`.
- [ ] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [ ] You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- [ ] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/model/crm14/Crm14Model.java [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import uk.gov.justice.laa.crime.equinity.historicaldata.model.CrmEvidenceFileModel;
 import uk.gov.justice.laa.crime.equinity.historicaldata.model.CrmEvidenceFilesModel;
 import uk.gov.justice.laa.crime.equinity.historicaldata.model.CrmFormModelInterface;
+import uk.gov.justice.laa.crime.equinity.historicaldata.model.data.Crm14AttachmentModel;
```
Comment (tanovellino) [inline equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/model/crm14/Crm14Model.java:10]: We should consider moving function `public void addProcessedAttachmentsToEvidence(...)` to class `CrmEvidenceFilesModel` which would eliminate the extra dependency here

Comment (solomonmanik) [reply equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/model/crm14/Crm14Model.java:10]: shall we do it in this PR ?

Comment (tanovellino) [reply equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/model/crm14/Crm14Model.java:10]: nope

Comment (tanovellino) [reply equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/model/crm14/Crm14Model.java:10]: added a TODO comment to check on the future

**Review by solomonmanik — APPROVED** [review]

**Pull request merged**